websurfx/config.lua

```diff
@@ -16,9 +16,8 @@ rate_limiter = {
 	number_of_requests = 20, -- The number of request that are allowed within a provided time limit.
 	time_limit = 3, -- The time limit in which the quantity of requests that should be accepted.
 }
--- Set whether the server will use an adaptive/dynamic HTTPS window size, see https://httpwg.org/specs/rfc9113.html#fc-principles
```

Comment:
I would suggest re-adding this explanation for the `https_adaptive_window_size` config option as it will be important for users to understand how and why to configure this option.

Reply:
@KekmaTime Thanks ❤️ for working on the suggestions and putting some new cool work in this PR. But we would suggest you to also clear this last small suggestion before we can merge this PR in. 🙂 .

Reply:
@KekmaTime Any updates on this PR so far? Would love to hear :slightly_smiling_face: .

Reply:
Since this requested was not resolved and the PR was open for a long time. So we have now resolved this change from our side.I hope you will not mind it. :slightly_smiling_face: .